src/annotator/test/guest-test.js

```diff
@@ -672,26 +676,51 @@ describe('Guest', () => {
       assert.calledWith(hostRPC().call, 'textUnselected');
     });
 
-    it('unselects text if another iframe has made a selection', () => {
-      const removeAllRanges = sandbox.stub();
-      sandbox.stub(document, 'getSelection').returns({ removeAllRanges });
-      const guest = createGuest();
-      guest.selectedRanges = [1];
+    context('on "clearSelection" RPC event', () => {
+      it('if guest has selected text, clears the selection', () => {
+        const removeAllRanges = sandbox.stub();
+        sandbox.stub(document, 'getSelection').returns({ removeAllRanges });
+        const guest = createGuest();
+        guest.selectedRanges = [1];
 
-      simulateSelectionWithText();
-      hostRPC().call.resetHistory();
-      emitHostEvent('clearSelection');
+        // Guest has text selected
+        simulateSelectionWithText();
+        fakeSelectedRange.returns({});
 
```

Comment:
I noticed a typo in the existing "should be inform the host" comment below. Should say "should inform the host". GitHub only changed code or code near changed code to be commented on during reviews.